podcast/the-changelog-438.md

```diff
@@ -218,7 +218,7 @@ Protocol Labs, Juan Benet who started the company - he's always had a lot of vis
 
 **Mikeal Rogers:** Yeah, so since we're talking about NFTs, the best way to look at it is to skip over some of the more file systemy aspects of it, and just think of it as addressing and availability in a peer-to-peer network.
 
-So if you give it some data, then it's gonna be able to give you back a content ID, which is a hash with some other data in front of it, that tells you what you can do with that hash, and what kind of format it is. Again, we've really thought about future-proofing these technologies against future hash functions, and different use cases... So you get this address, and you have an URL scheme where you can actually even path into that address... So you have \[unintelligible 00:29:30.02\] When you do check-ins with Git, you get that hash back, right? And that hash is immutably the state of the tree at that time... That's what you get out of IPFS - you get the immutable state of that tree, at that time. And that's really important, because in the IPFSnetwork, which is this global peer-to-peer network for getting the content, you can say "Hey, who has this address?" And you can find anybody on the internet with that address, and you can pull it from them, and in a trustless manner you can validate that that content is the content that they said that it was, right?
+So if you give it some data, then it's gonna be able to give you back a content ID, which is a hash with some other data in front of it, that tells you what you can do with that hash, and what kind of format it is. Again, we've really thought about future-proofing these technologies against future hash functions, and different use cases... So you get this address, and you have an URL scheme where you can actually even path into that address... So you have like have ipfs:// that address and then you can path it into it and that is effectively a tree. When you do check-ins with Git, you get that hash back, right? And that hash is immutably the state of the tree at that time... That's what you get out of IPFS - you get the immutable state of that tree, at that time. And that's really important, because in the IPFSnetwork, which is this global peer-to-peer network for getting the content, you can say "Hey, who has this address?" And you can find anybody on the internet with that address, and you can pull it from them, and in a trustless manner you can validate that that content is the content that they said that it was, right?
```

Comment:
```suggestion
So if you give it some data, then it's gonna be able to give you back a content ID, which is a hash with some other data in front of it, that tells you what you can do with that hash, and what kind of format it is. Again, we've really thought about future-proofing these technologies against future hash functions, and different use cases... So you get this address, and you have an URL scheme where you can actually even path into that address... So you have like have `ipfs://` that address and then you can path it into it and that is effectively a tree. When you do check-ins with Git, you get that hash back, right? And that hash is immutably the state of the tree at that time... That's what you get out of IPFS - you get the immutable state of that tree, at that time. And that's really important, because in the IPFSnetwork, which is this global peer-to-peer network for getting the content, you can say "Hey, who has this address?" And you can find anybody on the internet with that address, and you can pull it from them, and in a trustless manner you can validate that that content is the content that they said that it was, right?
```